src/pages/List.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { FirestoreCollection } from 'react-firestore';
+import Loading from '../components/Loading';
+
+const List = () => {
+  return (
+    <section>
+      <FirestoreCollection
+        // Specify the path to the collection you're pulling data from
+        path="items"
+        // Sort the data
+        sort="name"
+        // isLoading = is a Boolean that represents the loading status for the firebase query. true until an initial payload from Firestore is received.
```

Comment:
As earlier, are these comments to be breadcrumbs for the next team picking up the next issue?

Reply:
Yep! ❤️